multimodal/src/autogluon/multimodal/optimization/lit_distiller.py

```diff
@@ -199,6 +222,39 @@ def _compute_soft_label_loss(
         )
         return loss
 
+    def _compute_output_feature_loss(
```

Comment:
This function isn't called in `_compute_loss()`?

Reply:
Those lines are deleted by mistake. Added them back.